.asf.yaml

```diff
@@ -0,0 +1,22 @@
+github:
+  description: "Apache NuttX is a mature, real-time embedded operating system (RTOS)"
+  homepage: https://nuttx.apache.org/
+  features:
+    # Enable issues managment
+    issues: true
+    # Enable project for project managment boards
+    projects: true
+  labels:
+    - nuttx
+    - rtos
+    - embedded
+    - real-time
+    - mcu
+    - microcontroller
+  enabled_merge_buttons:
+    # enable squash button:
+    squash:  true
+    # enable merge button:
+    merge:   true
+    # disable rebase button:
+    rebase:  true
```

Comment:
Can we change rebase as the default option?

Reply:
Unfortunately you cannot set a default even in the GitHub UI so we can only enable/disable.
![image](https://user-images.githubusercontent.com/173245/72128706-a0efcd00-3328-11ea-8cf9-123a6197aa1a.png)


I would really like to see rebase used exclusively, but I think that should be handled in the workflow discussion rather than here so we can get issue tracking going. I do think the UI remembers what you last used, but I might be wrong.